src/sql/workbench/contrib/welcome/gettingStarted/browser/initialSetupWizard.ts

```diff
@@ -0,0 +1,452 @@
+				previousBtnContainer.classList.add('ads-initial-setup-wizard-previous-btn-container');
+				previousBtnContainer.classList.add('btn-secondary');
+				previousButton.label = localize('initialSetupWizard.previous', "Previous");
+				btnContainer.appendChild(previousBtnContainer);
+			}
+
+			btnContainer.appendChild(nextBtnContainer);
+			navContainer.appendChild(btnContainer);
+
+			if (interaction === 'checkbox') {
+				const interactionContainer: HTMLDivElement = document.createElement('div');
+				const label: HTMLLabelElement = document.createElement('label');
+				const checkbox: HTMLInputElement = document.createElement('input');
+				const span: HTMLSpanElement = document.createElement('span');
+				const p: HTMLParagraphElement = document.createElement('p');
+
+				p.innerText = interactionLabel;
+				checkbox.setAttribute('type', 'checkbox');
+				checkbox.setAttribute('name', name);
+
+				interactionContainer.classList.add('ads-initial-setup-wizard-checkbox-container');
+				label.classList.add('ads-initial-setup-wizard-switch');
+				checkbox.classList.add('ads-initial-setup-wizard-checkbox');
+				span.classList.add('ads-initial-setup-wizard-slider');
+				span.classList.add('round');
+
+				label.appendChild(checkbox);
+				label.appendChild(span);
+
+				interactionContainer.appendChild(label);
+				interactionContainer.appendChild(p);
+				textContainer.appendChild(interactionContainer);
+
+				checkbox.checked = true;
+
+				dom.addStandardDisposableListener(interactionContainer, 'keydown', event => {
+					if (event.equals(KeyCode.Enter)) {
+						if (checkbox.checked) {
+							checkbox.checked = false;
+						} else {
+							checkbox.checked = true;
+						}
+					}
+				});
+			}
+			if (interaction === 'button') {
+				const interactionContainer: HTMLDivElement = document.createElement('div');
+				interactionContainer.classList.add('ads-initial-setup-wizard-add-account');
+				interactionContainer.classList.add('ads-initial-setup-wizard-interaction-container');
+				const button: Button = new Button(interactionContainer);
+				button.label = interactionLabel;
+				button.onDidClick(() => {
+					// add account
+				});
+				textContainer.appendChild(interactionContainer);
+			}
+
+			contentContainer.appendChild(textContainer);
+			contentContainer.appendChild(hrTag);
+			contentContainer.appendChild(navContainer);
+			container.style.position = 'absolute';
+			container.appendChild(flexContainer);
+			this._container.append(container, $(`.${elmClass}`));
+			initialSetupElements.push(container);
+		});
+		this.buildInteractions();
+	}
+
+	private wizardNextEvent(popups: NodeListOf<Element>, popupsLength: number, i: number): void {
+		if (i === (popupsLength - 1)) {
+			this.hide();
+			return;
+		}
+		let next: number = i + 1;
+		const h1: HTMLElement = popups[next].querySelector('.ads-initial-setup-wizard-popup-content-container h1');
+		popups[i].classList.add('ads-initial-setup-wizard-hide');
```

Comment:
Am I missing something or is this class never defined anywhere?

And regardless - I'm not really sure I like the pattern here. Instead of showing/hiding "popups" why isn't this just adding/removing the elements as needed? We definitely shouldn't be keeping this stuff around in the DOM after the wizard is closed at the very least.